src/libraries/Microsoft.PowerFx.Core/Public/Config/PowerFxConfig.cs

```diff
@@ -46,15 +46,33 @@ private PowerFxConfig(CultureInfo cultureInfo, EnumStoreBuilder enumStoreBuilder
             EnumStoreBuilder = enumStoreBuilder;
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PowerFxConfig"/> class.        
+        /// </summary>
+        /// <param name="cultureInfo">Culture to use.</param>      
+        public PowerFxConfig(CultureInfo cultureInfo = null)
+            : this(cultureInfo, Features.None)
+        {
+        }
+
         /// <summary>
         /// Initializes a new instance of the <see cref="PowerFxConfig"/> class.        
         /// </summary>
         /// <param name="cultureInfo">Culture to use.</param>      
         /// <param name="features">Features to use.</param>
-        public PowerFxConfig(CultureInfo cultureInfo = null, Features features = Features.None)
-            : this(cultureInfo, new EnumStoreBuilder().WithRequiredEnums(BuiltinFunctionsCore.BuiltinFunctionsLibrary), features) 
+        public PowerFxConfig(CultureInfo cultureInfo, Features features)
+            : this(cultureInfo, new EnumStoreBuilder().WithRequiredEnums(BuiltinFunctionsCore.BuiltinFunctionsLibrary), features)
         {
-        }
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PowerFxConfig"/> class.        
+        /// </summary>
+        /// <param name="features">Features to use.</param>
+        public PowerFxConfig(Features features)
```

Comment:
Needed this overload because:
- required params can't come after optional params 
- we had callers just passing {features}